Assets/Scripts/Data Model/LayerManager.cs

```diff
@@ -1,948 +1,950 @@
+				}
 			}
 		}
-	}
 
-	public static SubEntity GetSubEntity(AbstractLayer layer, int subEntityID)
-	{
-		for (int i = 0; i < layer.GetEntityCount(); i++)
+		public static SubEntity GetSubEntity(AbstractLayer layer, int subEntityID)
 		{
-			Entity entity = layer.GetEntity(i);
-
-			for (int j = 0; j < entity.GetSubEntityCount(); j++)
+			for (int i = 0; i < layer.GetEntityCount(); i++)
 			{
-				SubEntity tmpEntity = entity.GetSubEntity(j);
-				if (tmpEntity.HasDatabaseID())
+				Entity entity = layer.GetEntity(i);
+
+				for (int j = 0; j < entity.GetSubEntityCount(); j++)
 				{
-					if (tmpEntity.GetDatabaseID() == subEntityID)
+					SubEntity tmpEntity = entity.GetSubEntity(j);
+					if (tmpEntity.HasDatabaseID())
 					{
-						return tmpEntity;
+						if (tmpEntity.GetDatabaseID() == subEntityID)
+						{
+							return tmpEntity;
+						}
 					}
 				}
 			}
-		}
 
-		return null; // This means it it doesnt exist
-	}
-
-	public static void ReorderLayers()
-	{
-		List<AbstractLayer> layersInOrder = GetLoadedLayersSortedByDepth();
-		layersInOrder.Reverse();
+			return null; // This means it it doesnt exist
+		}
 
-		for (int i = 0; i < layersInOrder.Count; i++)
+		public static void ReorderLayers()
 		{
-			layersInOrder[i].Order = i;
-			layersInOrder[i].LayerGameObject.transform.position = new Vector3(0, 0, -layersInOrder[i].Order);
-		}
-	}
+			List<AbstractLayer> layersInOrder = GetLoadedLayersSortedByDepth();
+			layersInOrder.Reverse();
 
-	private static void moveGeometry(AbstractLayer from, AbstractLayer to, int offset)
-	{
-		NetworkForm form = new NetworkForm();
+			for (int i = 0; i < layersInOrder.Count; i++)
+			{
+				layersInOrder[i].Order = i;
+				layersInOrder[i].LayerGameObject.transform.position = new Vector3(0, 0, -layersInOrder[i].Order);
+			}
+		}
 
-		form.AddField("old", from.ID);
-		form.AddField("new", to.ID);
-		form.AddField("offset", offset);
+		private static void moveGeometry(AbstractLayer from, AbstractLayer to, int offset)
+		{
+			NetworkForm form = new NetworkForm();
 
-		ServerCommunication.DoRequest(Server.MergeLayer(), form);
-	}
+			form.AddField("old", from.ID);
+			form.AddField("new", to.ID);
+			form.AddField("offset", offset);
 
-	public delegate void AddNewLayerCallback(AbstractLayer layer);
+			ServerCommunication.DoRequest(Server.MergeLayer(), form);
+		}
 
-	//public static void AddNewLayer(string name, GeoType type, AddNewLayerCallback addNewLayerCallback)
-	//{
-	//	NetworkForm form = new NetworkForm();
+		public delegate void AddNewLayerCallback(AbstractLayer layer);
 
-	//	form.AddField("name", name);
-	//	form.AddField("geotype", type.ToString());
+		//public static void AddNewLayer(string name, GeoType type, AddNewLayerCallback addNewLayerCallback)
+		//{
+		//	NetworkForm form = new NetworkForm();
 
-	//	ServerCommunication.DoRequest(Server.NewLayer(), form, (www2) => { handleNewLayerCallback(www2, addNewLayerCallback); });
-	//}
+		//	form.AddField("name", name);
```

Comment:
Commented code, as always see if it makes sense to leave it there for the future, otherwise remove it! :)